### PR TITLE
Fixed issue #17883: QueXML formatting cannot be changed

### PR DESCRIPTION
**Pull request by adamzammit**
Fixed issue #17883: QueXML formatting cannot be changed

### CODE REVIEW

**Comment by gabrieljenik** [comment]
Is this related to 17833?
https://bugs.limesurvey.org/view.php?id=17833

**Comment by adamzammit** [comment]
Apologies there was a typo. I've updated the comment to reflect the correct bug: #17883

File: application/controllers/admin/export.php [hunk 1]
```diff
@@ -1113,7 +1086,10 @@ private function _quexmlsettings()
      */
     public function quexmlclear($iSurveyID)
     {
-        $queXMLSettings = $this->_quexmlsettings();
+        Yii::import("application.libraries.admin.quexmlpdf", true);
+        $defaultquexmlpdf = new quexmlpdf($this->getController());
```
Comment (gabrieljenik) [inline application/controllers/admin/export.php:1090]: Does this constructor take an argument?

**Review by gabrieljenik — APPROVED** [review]
Code seems OK.
I believe for master we could turn the settings form into a screen of its own.

**Pull request merged**